wagtail/admin/templatetags/wagtailadmin_tags.py

```diff
@@ -1079,3 +1080,14 @@ def human_readable_date(date, description=None):
         "date": date,
         "description": description,
     }
+
+
+@register.simple_tag(takes_context=True)
+def get_edit_url(context, object):
+    user = None
+    request = getattr(context, "request", None)
+    if request:
+        user = request.user
+
+    url_finder = AdminURLFinder(user)
```

Comment:
I'm not really keen on us instantiating an `AdminURLFinder` on every item in the list - that's a lot of redundant reconstruction of lookup tables. More generally, I think creating a template tag for this is going to encourage more inefficient usage like this... Can we do the attaching of edit_url inside the view instead (using a single AdminURLFinder instance) and pass a tuple of (edit_url, reference) to the template?